src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -410,6 +410,12 @@ private async Task<(long, DatabaseTopology, List<string>)> CreateDatabase(string
                 Server.ServerStore.AssignNodesToDatabase(clusterTopology, databaseRecord);
             }
 
+            if (databaseRecord.SupportedFeatures == null || databaseRecord.SupportedFeatures.Count == 0)
```

Comment:
I wonder why the client need option to set it? 

I think there should be some method in `SupportedFeatureTypes` class that we will call here to populate `databaseRecord.SupportedFeatures` instead of doing `databaseRecord.SupportedFeatures.Add()` on each type

Reply:
done